wsagent/che-core-git-impl-jgit/src/main/java/org/eclipse/che/git/impl/jgit/JGitConnection.java

```diff
@@ -308,6 +310,8 @@
   private final GitUserResolver userResolver;
   private final Repository repository;
 
+  private static boolean commitInProgress;
```

Comment:
In case of several projects we need to have a `Set`

Reply:
done